docs/aerodromes/classc/Darwin.md

```diff
@@ -47,6 +47,31 @@ d) Runway 11 and Runway 36;
 e) Runway 11 and Runway 18;  
 f) Runway 11 only.  
 
+## LAHSO
+
+LAHSO is the operation of two crossing runways used for arrivals and departures. Tower should give aircraft traffic information about the secondary aircraft landing on the crossing runway. Darwin utilises LAHSO in the runway format of 36A/29AD
+
+!!! phraseology
+    ***Traffic Landing RWY 36***
+
+    **DN ADC:** "YZU, traffic is an E120, landing on the crossing runway, hold short runway 29, runway 36 cleared to land."
```

Comment:
Incorrect phraseology. As per AIP; "Hold Short Runway XX. Cleared to Land Runway XX". This is the only time a runway comes AFTER a clearance to land or takeoff.

Also... remove the comma after traffic is a E120